gradle.properties

```diff
@@ -1,5 +1,5 @@
-kotlinVersion=1.0.6
-junitPlatformVersion=1.0.0-M3
+kotlinVersion=1.1.1
```

Comment:
Can you revert this to `1.0.6`? I know it's very minor, but I prefer commits to do single change.